Automattic/wp-calypso
# Global Styles: add more fonts to list

This PR adds more fonts and pairings, as well as handling fonts that are deprecated.

**TODO:**

- [ ] Design for pairings controls.

## Testing instructions

Install the FSE plugin locally:

* `npm install`
* `npx lerna run build --scope='@automattic/full-site-editing'`
* Move the `<local-path-to-calypso>/apps/full-site-editing/full-site-editing-plugin/` under  `wp-content/plugins` in your local WordPress install.

Install a theme that uses Global Styles:

* Clone the [themes](https://github.com/automattic/themes) repo locally.
* `cd varia`
* `npm install`
* `npm run build`
* Move `<local-path-to-themes>/varia` under `wp-content/themes` in your local WordPress install.

Test:

* Activate the Full Site Editing plugin and the Varia theme
* Go to a post.
* Open the Global Styles sidebar and verify that fonts work.
